extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -0,0 +1,260 @@
+          "type": "string",
+          "meta:enum": {
+            "Undefined": "Not defined",
+            "UnknownUser": "User unknown",
+            "InvalidDomain": "Invalid domain",
+            "Unreachable": "Unreachable",
+            "Disabled": "Account disabled",
+            "MailboxFull": "Mailbox full",
+            "NotConnected": "Not connected",
+            "Refused": "Refused",
+            "ErrorIgnored": "Error ignored",
+            "AddressUndefined": "Address not specified",
+            "Blacklisted": "Blacklisted address",
+            "Quarantine": "Quarantined address",
+            "Duplicate": "Duplicate",
+            "TypologyRule": "Excluded by a SQL rule",
+            "BusinessRanking": "Excluded after arbitration",
+            "Cancelled": "Delivery canceled",
+            "Quality": "Bad-quality address",
+            "Unchecked": "Unqualified address",
+            "OfferMissing": "Not eligible for the offers",
+            "OverDeliveryLimit": "Target limited in size",
+            "Expired": "Expired",
+            "TooLong": "Text too long",
+            "Untranslatable": "Character not supported by encoding",
+            "ControlGroup": "Control address"
+          },
+          "description":
+            "The reason why the message could/would not be delivered.",
+          "default": "Undefined"
+        },
+        "https://ns.adobe.com/experience/campaign/message/reasonMessage": {
+          "title": "Reason Message",
+          "type": "string",
+          "description":
+            "Reason of message delivery failure when the status is `bounce` or `nonDeliverable`."
+        },
+        "https://ns.adobe.com/experience/campaign/message/outboundIP": {
+          "title": "IP Address",
+          "type": "string",
+          "oneOf": [
+            {
+              "format": "ipv4"
+            },
+            {
+              "format": "ipv6"
+            }
+          ],
+          "description": "Outbound IP address used to deliver the message."
+        },
+        "https://ns.adobe.com/experience/campaign/delivery/id": {
+          "title": "DeliveryID",
+          "type": "integer",
+          "description": "The campaign activity originating this message."
+        },
+        "https://ns.adobe.com/experience/campaign/delivery/channel": {
+          "title": "Communication Channel",
+          "type": "string",
+          "meta:enum": {
+            "https://ns.adobe.com/xdm/channels/email": "Email",
+            "https://ns.adobe.com/xdm/channels/sms": "Mobile (SMS)",
+            "https://ns.adobe.com/xdm/channels/phone": "Phone",
+            "https://ns.adobe.com/xdm/channels/direct-mail": "Direct Mail",
+            "https://ns.adobe.com/xdm/channels/fax": "Fax",
+            "https://ns.adobe.com/xdm/channels/apns":
+              "Apple Push Notification Service",
+            "https://ns.adobe.com/xdm/channels/gcm": "Google Cloud Messaging",
+            "agency": "Agency",
```

Comment:
Introducing this as a proper channel (URL, schema) would make sense.

Reply:
:+1: